docs/RFCS/range_leases.md

```diff
@@ -137,3 +142,7 @@ range leases using the proposed system.
 
 How does this mechanism inform future designs to incorporate quorum
 leases?
+
+TODO(peter): What is the motivation for gossipping the node lease
```

Comment:
It's out of date, but it's cheap. The idea is that by gossiping this table, once the lease of a down node has been revoked, all the nodes will see it quickly without have to do their own polling of the lease table. There is still some room for a thundering herd effect when a node goes down, but this will minimize how many nodes have to attempt revocation and touch the lease table.